x-pack/plugins/actions/server/plugin.ts

```diff
@@ -351,6 +351,7 @@ export class ActionsPlugin implements Plugin<PluginSetupContract, PluginStartCon
         actionType: ActionType<Config, Secrets, Params, ExecutorResultData>
       ) => {
         ensureSufficientLicense(actionType);
+        actionType.maxAttempts = actionType.maxAttempts ?? 3;
```

Comment:
Can we keep this magic number (3) as a constant?
As far as i can see, it's used in 3 different places.

Reply:
Sure, I can do that!

Reply:
Resolved in https://github.com/elastic/kibana/pull/143224/commits/b55c166ca339d51eb4129c675fa0f1cfce10f504